docs/en/source/callFlow.rst

```diff
@@ -175,33 +170,35 @@ Context Source Subscription
 
 2. The request enters in service API gateway.
 
- a. The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
+ 2.1 The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
+
+ 2.2 The service API gateway forwards the HTTP request to the Context Source Registry micro-service.
 
- b.The service API gateway forwards the HTTP request to the Context Registry (CR) Manager micro-service.
-   
-3. The CR manager now fetches the previously stored data/entities from the Topic “CSourceSub”.
+3. The Context Source Registry now fetches the previously stored data/entities from the Topic “CSourceSub”.
 
- a. Now the CR manager performs some basic validation to check if this is a valid request with the valid payload. 
+- Now the Context Source Registry performs some basic validation to check if this is a valid request with the valid payload.
 
- b.If the entry for the request csource subscription is already present it exits the processing and informing the same to the requester. If it is not present, then it continues for further processing.
+- If the entry for the request csource subscription is already present it exits the processing and informing the same to the requester. If it is not present, then it continues for further processing.
 
- c.CR manager now writes this payload into the CsourceSub Topic.
+- Context Source Registry now writes this payload into the CSourceSub Topic.
 
- d.In parallel, it will also start an independent thread to listen Csource Topic for the requested subscription and upon the successful condition, the notification will be sent to the registered endpoint provided under subscription payload.
+- In parallel, it will also start an independent thread to listen Csource Topic for the requested subscription and upon the successful condition, the notification will be sent to the registered endpoint provided under subscription payload.
+ 
+4. Now, our csource subscription request firstly gets publish/store in the database and after that further processing is done in the Kafka bus.
 
-4. The Storage Manager will keep listening for the CsourceSub topic and for any new/updated entry write it perform the relative operation in the database. 
+5. Registry Subscription Manager will keep listening to the CSourceSub topic and for any new entry.
 
-5. The CR manager prepares the response for csource subscription request and
+6. The Registry Subscription manager prepares the response for csource subscription request and
 
- 5.1 sends the Http response back to the API gateway.
+ 6.1 sends the Http response back to the API gateway.
 
- 5.2 API gateway sends back the response to the end-user/requester.
+ 6.2 API gateway sends back the response to the end-user/requester.
```

Comment:
[Major] Please changes in flow diagram 5
- Remove topic direction same as above
-  Add direction from registry subscription manager to  I_NOTIFY topic
-  Add direction from  I_NOTIFY topic to subscription manager
-  Changes in theoretical part same as flow diagram direction.

Reply:
Done